rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -170,22 +201,8 @@ byte[] sendRequest(HttpSender.Request request) throws IOException, HttpSenderRes
         }
     }
 
-    private Map<GroupArtifactVersion, Pom> projectPomsByGav(Map<Path, Pom> projectPoms) {
-        Map<GroupArtifactVersion, Pom> result = new HashMap<>();
-        for (Pom projectPom : projectPoms.values()) {
-            List<Pom> ancestryWithinProject = getAncestryWithinProject(projectPom, projectPoms);
-            Map<String, String> mergedProperties = mergeProperties(ancestryWithinProject);
-            GroupArtifactVersion gav = new GroupArtifactVersion(
-                    projectPom.getGroupId(),
-                    projectPom.getArtifactId(),
-                    ResolvedPom.placeholderHelper.replacePlaceholders(projectPom.getVersion(), mergedProperties::get)
-            );
-            result.put(gav, projectPom);
-        }
-        return result;
-    }
 
-    private Map<String, String> mergeProperties(final List<Pom> pomAncestry) {
+    public static Map<String, String> mergeProperties(final List<Pom> pomAncestry) {
```

Comment:
Should these methods perhaps move to the MavenExecutionContextView? Then on line 507 you can cast the context and call it that way

Reply:
I think you are right. Let me move those.